src/App.tsx

```diff
@@ -34,62 +36,116 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return good1.localeCompare(good2);
+      case SortType.LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
```

Comment:
I think it would be more "natural" to choose not sort callback but sort expression itself.
And it helps to avoid weird `default: return 0;`
Plus simple `visibleGoods.sort();` is enough for alphabetical sorting (without `localeCompare`)

```suggestion
  switch (sortType) {
    case SortType.ALPHABET:
      return visibleGoods.sort();
      break;
    case SortType.LENGTH:
      return visibleGoods.sort((good1, good2) => good1.length - good2.length);
      break;
    default:
      break;
    }
```